tutorials/pyMBE_tutorial.ipynb

```diff
@@ -231,6 +227,28 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "# Only necesary to produce the pictures used in this tutorial\n",
+    "from PIL import Image\n",
+    "def do_snapshot_espresso_system(espresso_system, filename):\n",
```

Comment:
Just a nomenclature thing, but I would call this function `create_snapshot_espresso_system`